### ISSUE TITLE
Can you provide the RD3D results of ReDWeb-S?

### ISSUE THREAD

**Issue by fereenwong** [PPOLYpubki/RD3D#10]
Hi, thanks for sharing.
Can you provide the RD3D results of ReDWeb-S?

**Comment by PPOLYpubki** [comment]
ok, the RD3D results of ReDWeb-S have been updated～

**Comment by fereenwong** [comment]
Thanks a lot. I download and evaluate the updated RD3D results of ReDWeb-S, it results in a performance that differs from the one listed in Table. (RD3D: ReDWeb-S 0.700). Are the updated results consistent with the one used to report the performance?

**Comment by PPOLYpubki** [comment]
Yes, since we did not retrain the ReDWeb-S dataset and only tested it on this dataset with the previous model, this result is to be expected.



At 2021-11-23 14:57:52, "fereenwong" ***@***.***> wrote:

Thanks a lot. I download and evaluate the updated RD3D results of ReDWeb-S, it results in a performance that differs from the one listed in Table. (RD3D: ReDWeb-S 0.700). Are the updated results consistent with the one used to report the performance?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by fereenwong** [comment]
Sorry for the misunderstanding. What I want is the results of the previous models.  As I want to add this as a comparison method in my paper, which keeps the same setting as the previous models (not trained with the trainset of ReDWeb-S). I tried to get it with the provided pretrained model, maybe I did something wrong, I can not exactly reproduce the performance as reported. Could you please send me a copy of the ReDWeb-S results with previous models? Email: fereenwong@gmail.com. This will help me a lot. Thanks again.
